code/modules/mob/living/carbon/human/species.dm

```diff
@@ -637,6 +637,40 @@ GLOBAL_LIST_EMPTY(roundstart_races)
 				eye_overlay.pixel_y += H.dna.species.offset_features[OFFSET_FACE][2]
 			standing += eye_overlay
 
+	// organic body markings
+	if(HAS_MARKINGS in species_traits)
+		var/obj/item/bodypart/chest/CH = H.get_bodypart(BODY_ZONE_CHEST)
+		var/obj/item/bodypart/r_arm/RA = H.get_bodypart(BODY_ZONE_R_ARM)
+		var/obj/item/bodypart/l_arm/LA = H.get_bodypart(BODY_ZONE_L_ARM)
+		var/obj/item/bodypart/r_leg/RL = H.get_bodypart(BODY_ZONE_R_LEG)
+		var/obj/item/bodypart/l_leg/LL = H.get_bodypart(BODY_ZONE_L_LEG)
+		var/datum/sprite_accessory/markings = GLOB.moth_markings_list[H.dna.features["moth_markings"]]
+		var/HUSKED = HAS_TRAIT(H, TRAIT_HUSK)
```

Comment:
1. This doesn't need to be in all caps--those are generally reserved for shorthand names (which we should stop using) and defines.

2. I'd rather you just wrap all the if statements underneath in an `if (!HAS_TRAIT(H, TRAIT_HUSK))` rather than making a variable to check every time.